fdbserver/BlobManager.actor.cpp

```diff
@@ -5042,6 +5048,26 @@ ACTOR Future<int64_t> bgccCheckGranule(Reference<BlobManagerData> bmData, KeyRan
 	return bytesRead;
 }
 
+// Check if there is any pending split. It's a precheck for manifest backup
+ACTOR Future<bool> hasPendingSplit(Reference<BlobManagerData> self) {
+	state Transaction tr(self->db);
+	loop {
+		tr.setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
```

Comment:
also add PRIORITY_SYSTEM_IMMEDIATE and LOCK_AWARE so that this transaction will run even if the cluster is throttled by ratekeeper, and even when the cluster is locked

Reply:
fixed. thank you!